libc/benchmarks/CMakeLists.txt

```diff
@@ -1,26 +1,33 @@
+if(LIBC_TARGET_OS_IS_GPU)
+  add_subdirectory(gpu)
+  return()
+endif()
+
```

Comment:
This file shouldn't have any changes except for these five lines.

Reply:
Yeah, I ended up tweaking some of my editor settings and I think I got it now